templates/test/by_id.html

```diff
@@ -67,7 +67,34 @@
                         </tbody>
                     </table>
                 {% else %}
-                    <p>This test is still in the queue! Please check again later.</p>
+                      <p>This test is still in the queue! Please check again later.Wait {{time}} minutes</p>
```

Comment:
{{ time }} is not containing the estimated time by when the test will be, so it's of no use to the user at all.